nixops/backends/digital_ocean.py

```diff
@@ -165,5 +164,6 @@ def create(self, defn, check, allow_reboot, allow_recreate):
         # - no reboot
         # - predictable network interface naming (enp0s3 etc)
         self.wait_for_ssh()
+        self.log_start("running modified nixos-infect")
```

Comment:
In order not to confuse users, I'd rather skip `modified` since it's already mentioned in comments. It's really a developer note rather than nixops user's.